department-of-veterans-affairs/caseflow
# Fixes Reader batch job when documents are cached in S3

Connects #3010
 
- Removes document.fetch_content_unless_cached()
    - Moves the functionality to the batch job since nothing else uses the function
- Clarifies the message sent to Slack about the job success
- Adds the job name to the failure Slack message

To test:
- Run the job in the Rails console with the command `RetrieveDocumentsForReaderJob.perform_now("limit" => 1)` to test that it works with VBMS
- Enable the eFolder API feature flag in the Rails console, `FeatureToggle.enable!(:efolder_docs_api)`, and make sure the job makes the call to the eFolder `\files` endpoint
